transforms/trailing-commas.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Adds trailing commas to every object litteral and array.
+ */
+export default function(file, api, options) {
+  const j = api.jscodeshift;
+
+  const objectHasNoTrailingComma = ({node}) => {
+    // Only transform objects that are on multiple lines.
+    if (node.properties.length === 0 || node.loc.start.line === node.loc.end.line) {
+      return false;
+    }
+    const lastProp = node.properties[node.properties.length - 1];
+    return file.source.charAt(lastProp.end) !== ',';
+  };
+
+  const arrayHasNoTrailingComma = ({node}) => {
+    // Only transform arrays that are on multiple lines.
+    if (node.elements.length === 0 || node.loc.start.line === node.loc.end.line) {
+      return false;
+    }
+    const lastEle = node.elements[node.elements.length - 1];
+    return file.source.charAt(lastEle.end) !== ',';
+  };
+
+  const forceReprint = ({node}) => {
+    node.original = null;
+  };
+
+  const root = j(file.source);
+  root
+    .find(j.ObjectExpression)
+    .filter(objectHasNoTrailingComma)
+    .forEach(forceReprint);
+  root
+    .find(j.ArrayExpression)
+    .filter(arrayHasNoTrailingComma)
+    .forEach(forceReprint);
```

Comment:
Its normally worth grabbing the `.size()` here and the statement above and only doing `.toSource()` if they are both greater than `0`. This will make the code mod run over a large code base much quicker as well as save you from some weird recast reprinting errors the come up.